wrappers/javascript/indy-vdr-react-native/package.json

```diff
@@ -44,22 +44,30 @@
     "@mapbox/node-pre-gyp": "^1.0.10"
```

Comment:
FYI @bryce-mcmath, in the AnonCreds and Askar wrappers we replaced the dependency on node-pre-gyp with a custom install script. Since it would fail in React Native with PNPM. and the pre-gyp dep is quite heavy.

Might be nice to also integrate it into the indy wrapper: https://github.com/openwallet-foundation/askar-wrapper-javascript/pull/40